app.yaml

```diff
@@ -0,0 +1,8 @@
+env: standard
+runtime: nodejs16
+service: default
+
+handlers:
+  - url: /.*
+    secure: always
```

Comment:
When @tinpham5614 ran me through this process this morning, I had indented the properties under url:
Not sure if it will affect other's setups. Just a heads up.

Reply:
I think it’s a copy error. I fixed it in the guide.